newsletter/admin.py

```diff
@@ -43,13 +44,22 @@
 
 from .settings import newsletter_settings
 
+
+logger = logging.getLogger(__name__)
+
+
 # Contsruct URL's for icons
-ICON_URLS = {
-    'yes': '%snewsletter/admin/img/icon-yes.gif' % settings.STATIC_URL,
-    'wait': '%snewsletter/admin/img/waiting.gif' % settings.STATIC_URL,
-    'submit': '%snewsletter/admin/img/submitting.gif' % settings.STATIC_URL,
-    'no': '%snewsletter/admin/img/icon-no.gif' % settings.STATIC_URL
-}
+def icon_url(icon, alt_text, _urls={}):
+    if not _urls:
+        _urls['yes'] = static('newsletter/admin/img/icon-yes.gif')
+        _urls['wait'] = static('newsletter/admin/img/waiting.gif')
+        _urls['submit'] = static('newsletter/admin/img/submitting.gif')
+        _urls['no'] = static('newsletter/admin/img/icon-no.gif')
+
+    return format_html(
+        '<img src="{}" width="10" height="10" alt="{}"/>',
+        _urls[icon], alt_text
+    )
```

Comment:
> Thanks for the comment on the SVG. Are you sure the HTML to render them is the same? I have a vague memory of the dimensions being set through CSS or something.

The HTML is the same, you can see the [relevant code in Django master](https://github.com/django/django/blob/master/django/contrib/admin/templatetags/admin_list.py#L181) just outputs an `img` with an SVG.